packages/deployer/internal/ast-helper.js

```diff
@@ -0,0 +1,142 @@
+const path = require('path');
```

Comment:
I think that this is a good exercise to understand AST's produced by the Solidity compiler, but I think that we should be using a generic library instead of this. This is fragile, and too specific to the case at hand.